cpu/cortexm_common/Makefile.features

```diff
@@ -30,3 +30,8 @@ else ifeq ($(CPU_CORE),cortex-m23)
 else
   $(error Unkwnown cortexm core: $(CPU_CORE))
 endif
+
+# cortex-m3 and higher don't need the idle thread
+ifneq (,$(filter cortex-m2% cortex-m4% cortex-m3% cortex-m7%,$(CPU_ARCH)))
```

Comment:
```suggestion
ifneq (,$(filter cortex-m2% cortex-m4% cortex-m3% cortex-m7%,$(CPU_CORE)))
```

Reply:
On second thoughts, maybe checking for the architecture is cleaner?
```Makefile
ifneq (,$(filter armv7 armv8,$(CPU_ARCH)))
```

Reply:
I don't mind, personally I'm thinking "cortex-m3 and higher" instead of "armv7". your call.

Reply:
I would go with the arch approach just to be in line with what you added in the cortexm_common Kconfig.

Reply:
ok

Reply:
Oh I think the correct values are `armv7m` and `armv8m` :/

Reply:
:) fixed